cypress/e2e/awx/resources/hosts.cy.ts

```diff
@@ -37,86 +39,156 @@ describe('host and inventory host', () => {
+    cy.hasDetail(/^Variables$/, 'test: true');
 
-  if (inventory_host) {
-    cy.get('[data-cy="empty-state-title"]').contains(
-      /^There are currently no hosts added to this inventory./
-    );
+    return hostName;
   }
 
-  // create
-  cy.clickButton(/^Create host$/);
-  cy.verifyPageTitle('Create Host');
-  cy.get('[data-cy="name"]').type(hostName);
-  cy.get('[data-cy="description"]').type('This is the description');
-
-  if (!inventory_host) {
-    cy.get(`[data-cy="inventory-id"]`).click();
-    cy.contains('button', 'Browse').click();
-    cy.contains('Select Inventory');
-    cy.get(`[aria-label="Select Inventory"]`).within(() => {
-      cy.filterTableBySingleSelect('name', inventory);
-      cy.get(`[data-cy="checkbox-column-cell"] input`).click();
-      cy.contains('button', 'Confirm').click();
-    });
-    cy.get(`[aria-label="Select Inventory"]`).should('not.exist');
+  function editHost(inventoryID: number, inventory_host: boolean, hostName: string) {
+    if (inventory_host) {
+      cy.visit(
+        `/infrastructure/inventories/inventory/${inventoryID}/hosts/?page=1&perPage=10&sort=name`
+      );
+    } else {
+      cy.visit('/infrastructure/hosts?page=1&perPage=10&sort=name');
+    }
+    cy.filterTableByMultiSelect('name', [hostName]);
+
+    cy.getByDataCy('edit-host').click();
+    cy.verifyPageTitle('Edit host');
+    cy.getByDataCy('description').clear().type('This is the description edited');
+    cy.getByDataCy('Submit').click();
+
+    cy.hasDetail(/^Description$/, 'This is the description edited');
+  }
+
+  function deleteHost(inventoryID: number, inventory_host: boolean, hostName: string) {
+    if (inventory_host) {
+      cy.visit(
+        `/infrastructure/inventories/inventory/${inventoryID}/hosts/?page=1&perPage=10&sort=name`
+      );
+    } else {
+      cy.visit('/infrastructure/hosts?page=1&perPage=10&sort=name');
+    }
+
+    cy.filterTableByMultiSelect('name', [hostName]);
+    cy.get(`[data-cy="actions-column-cell"] [data-cy="actions-dropdown"]`).click();
+    cy.getByDataCy('delete-host').click();
+    cy.clickModalConfirmCheckbox();
+    cy.clickModalButton('Delete hosts');
+    cy.contains('button', 'Close').click();
+    cy.contains(/^No results found./);
   }
 
-  cy.getByDataCy('variables').type('test: true');
-  cy.clickButton(/^Create host/);
-  cy.hasDetail(/^Name$/, hostName);
-  cy.hasDetail(/^Description$/, 'This is the description');
-  cy.hasDetail(/^Variables$/, 'test: true');
-
-  return hostName;
-}
-
-function editHost(inventoryID: number, inventory_host: boolean, hostName: string) {
-  if (inventory_host) {
-    cy.visit(
-      `/infrastructure/inventories/inventory/${inventoryID}/hosts/?page=1&perPage=10&sort=name`
-    );
-  } else {
-    cy.visit('/infrastructure/hosts?page=1&perPage=10&sort=name');
+  function navigateToHost(url: string, name: string, data: string) {
+    cy.visit(url);
+    cy.searchAndDisplayResource(name || '');
```

Comment:
Update it to use another function compatible with new search component

Reply:
done